hack/generate/samples/internal/go/memcached-with-customization/memcached_with_customization.go

```diff
@@ -39,25 +39,40 @@ var prometheusAPIVersion = "v0.59.0"
 // GenerateSample will call all actions to create the directory and generate the sample
 // Note that it should NOT be called in the e2e tests.
 func GenerateSample(binaryPath, samplesPath string) {
-	log.Infof("starting to generate Go memcached sample with webhooks and metrics documentation")
-	ctx, err := pkg.NewSampleContext(binaryPath, filepath.Join(samplesPath, "memcached-operator"), "GO111MODULE=on")
-	pkg.CheckError("generating Go memcached with webhooks and metrics documentation context", err)
-
 	generateWithMonitoring = false
 	if strings.HasSuffix(samplesPath, "monitoring") {
 		generateWithMonitoring = true
 	}
 
+	var logInfo, errorInfo string
+	if generateWithMonitoring {
+		logInfo = "starting to generate Go memcached sample with webhooks and monitoring"
+		errorInfo = "generating Go memcached with webhooks and monitoring context"
+	} else {
+		logInfo = "starting to generate Go memcached sample with webhooks"
+		errorInfo = "generating Go memcached with webhooks context"
+	}
```

Comment:
Non-blocking nit: I think we could probably simplify this logic with a better formatted log/error information to print. 

I'm thinking something along the lines of:
```go
logInfo := fmt.Sprintf("starting to generate Go memcached sample with webhooks and monitoring=%t", generateWithMonitoring)
errorInfo :=  fmt.Sprintf("generating Go memcached with context: webhooks and monitoring=%t", generateWithMonitoring)
```

Less conditional logic makes it easier to read and understand IMO.

Reply:
I update it slightly different then you suggested but I think it's better, can you take a look?